### DISCUSSION TITLE
Reset specific vehicles rather than all spawned vehicles

### DISCUSSION THREAD

**Discussion by SaundersJE97** [microsoft/AirSim#4173] [Support Q&A]
When I use the client.reset() function, this resets all spawned vehicles.  Is there a way to specify the vehicle to be reset rather than to reset all vehicles?  

I am attempting to reset the collision info 'has_collided' but I am currently have to use a work around by using the minimum distance value of the depth camera.

**Comment by jonyMarino** [comment]
Hi @SaundersJE97! it seems a good feature request to have a `vehicle_name` argument for the `reset()` method

**Reply by SaundersJE97** [reply]
I agree
